src/XrdOss/XrdOssStat.cc

```diff
@@ -123,7 +123,7 @@ int XrdOssSys::Stat(const char *path, struct stat *buff, int opts,
 //
    if ((retc = MSS_Stat(remote_path, buff))) return retc;
    if (popts & XRDEXP_NOTRW) buff->st_mode &= ro_Mode;
-   buff->st_mode |= S_IFBLK;
+   // buff->st_mode |= S_IFBLK;
```

Comment:
Frankly, this should be removed. If you want to leave it in, a comment would be necessary on why this is commented out. I agree that the flag should not have been set and full removal avoids provoking questions.

Reply:
I've removed it.